tutorials/publish/index.md

```diff
@@ -349,6 +349,34 @@ It also should be noted that if your etcd storage is safe and channels have not
 
 Our recommendations are following
 - Your should run ```snet treasurer claim-expired``` each 1-3 days. We recommend automate it using `cron`.
-- You can run ```claim-all``` command when your want. For example once in serveral month.
+- You can run ```claim-all``` command when your want. For example once in several months.
 
-For more information about the `SNET MultiPartyEscrow` check this [link](https://github.com/singnet/dev-portal/tree/master/docs/all/mpe). 
+For more information about the `SNET MultiPartyEscrow` check this [link](https://dev.singularitynet.io/docs/all/mpe/mpe/). 
+
+## Step 11 (optional). Withdraw AGI tokens from MPE
+
+After the step 10, all the AGIs are in the MultiPartyEscrow Contract (MPE).
+
+If you need to transfer them to your wallet, you must run the `SNET CLI` withdraw command.
```

Comment:
I'm not sure that "wallet" is a right term. Technically MPE is a wallet. I would simply say "If you need to withdraw you tokens from MPE wallet, you should run the `SNET CLI` withdraw command"

Reply:
Ok.